Orange/projection/manifold.py

```diff
@@ -190,6 +182,23 @@ def __init__(self, n_components=2, affinity='nearest_neighbors', gamma=None,
         self.params = vars()
 
 
+class lazy_openTSNE:
+    """openTSNE uses numba, which is slow to load, so load it lazily."""
+    def __getattr__(self, item):
+        import sys
+        import openTSNE
+        import openTSNE.affinity
+        import openTSNE.initialization
+        if "openTSNE" in sys.modules:
```

Comment:
@pavlin-policar, I'm trying to revive this, but since the change caused a crash on travis in bioinformatics, it is difficult to debug.

Lookin for a cause, I found this condition very strange. When you import a module, it always appears in `sys.modules`, doesn't it? Why did you need this line, what did it try to prevent, what was the case when the condition was false?

Have you by any chance tried adding `global openTSNE` to the function?

Is it possible that bioinformatics tried to do something in parallel and this function was executed in parallel (though I don't see how this could have happened), and numba initialized twice? Qt, for instance, does (or used to) crash if you initialized QApplication twice.